packages/polymath-js/src/contracts/Contract.js

```diff
@@ -15,8 +15,17 @@ import type {
 // this has to be here for now
 const HARDCODED_NETWORK_ID = 15;
 
+const noRegistryAddressError = new Error(
```

Comment:
Just in case, not sure what happens here with the stack trace. If the stack trace is set when the error is instanced then it'd be better to just save the text in the constant and instance the new error where we throw it.